easybuild/tools/packaging.py

```diff
@@ -0,0 +1,116 @@
+# #
+# Copyright 2009-2014 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# the Hercules foundation (http://www.herculesstichting.be/in_English)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+# #
+
+"""
+A place for packaging functions
+
+@author: Marc Litherland
+@author: Gianluca Santarossa (Novartis)
+@author: Robert Schmidt (Ottawa Hospital Research Institute)
+@author: Fotis Georgatos (Uni.Lu, NTUA)
+@author: Kenneth Hoste (Ghent University)
+"""
+
+import os
+import tempfile
+import pprint
+
+from vsc.utils import fancylogger
+
+from easybuild.tools.run import run_cmd
+from easybuild.tools.config import install_path, package_template
+from easybuild.tools.module_naming_scheme.utilities import det_full_ec_version
+from easybuild.tools.toolchain import DUMMY_TOOLCHAIN_NAME
+
+_log = fancylogger.getLogger('tools.packaging')
+
+def package_fpm(easyblock, modfile_path, package_type="rpm" ):
```

Comment:
what if this was also tunable via `package_type`? L55/L113 could be similarly sync'ed to this.